resources/views/contacts/contactUs.blade.php

```diff
@@ -0,0 +1,92 @@
+@extends('layout')
+
+@section('pageTitle')
+    Contact us
+@endsection
+
+@section('content')
+    <div class="row">
+        <div class="col-lg-12 m-4">
+            <h4>Contact us</h4>
+            @if ($errors->any())
+                <div class="alert alert-danger">
+                    <ul>
+                        @foreach ($errors->all() as $error)
+                            <li>{{ $error }}</li>
+                        @endforeach
+                    </ul>
+                </div>
+            @endif
+            @if (session()->has('success'))
+                <div class="alert alert-success">
+                    {{ session('success') }}
+                </div>
+            @endif
+            <form action="{{ route('contactUs.send') }}" method="POST" name="contact-form">
+                <div class="form-row">
+                    <div class="form-group col-md-6">
+                        <label for="email">Email</label>
+                        <input type="email" value="{{ old('email') }}" required="required" class="form-control" name="email" id="email" placeholder="Email">
+                    </div>
+                    <div class="form-group col-md-6">
+                        <label for="name">Name</label>
+                        <input type="text" value="{{ old('name') }}" required="required" class="form-control" name="name" id="name" placeholder="Nume">
+                        @error('name')
```

Comment:
I think you can remove this one. You have the global one.

Reply:
I delete this.